Add page describing how to setup backtrace links. (#295)
appsignal/appsignal-docs: source/application/backtrace-links.html.md

---
title: "Backtrace links"
---

> With backtrace links you can spend less time figuring out where an exception happened and more time debugging the exception.

Reviewer: ```suggestion
With backtrace links you can spend less time figuring out where an exception happened and more time debugging the exception. Backtrace links make every line to source code in your app link to your app's source code on your source code hosting platform of choice.
```

> To enable backtrace links we need to:

Reviewer: ```suggestion
To enable backtrace links an app on AppSignal.com needs to:
```

> * [Enable deploy markers](http://localhost:4567/application/markers/deploy-markers.html).

Reviewer: localhost link 😅 

```suggestion
* [Enable deploy markers](/application/markers/deploy-markers.html).
```

> * Configure the "repo url" for an application, or link your organisation to GitHub.

Reviewer: ```suggestion
* Configure the "repo url" for an application, or [link your organization to GitHub](https://appsignal.com/redirect-to/organization?to=admin/integrations/github), so AppSignal knows where to link to.
```

We use [American English](https://github.com/appsignal/handbook/blob/master/general/communication.md#writing) in the docs.

## Configure the repo url

> Owners of an Organisation can specify a Repo url on [the app settings page](https://appsignal.com/redirect-to/app?to=edit).

Reviewer: I would mention this is specifically useful for other platforms than GitHub, for which we have the integration.

![App settings repo URL form](/assets/images/screenshots/repo_url.png)

> ## Link your organisation to GitHub.

Reviewer: ```suggestion
## Link your organisation to GitHub
```

> Alternatively organisation owners can [link the organisation](https://appsignal.com/redirect-to/app?to=integrations) to GitHub and we'll fill out the "repo url" from the selected repository automatically. You'll also be able to send incidents to GitHub directly from AppSignal.

Reviewer: ```suggestion
Alternatively organisation owners can [link an app to GitHub](https://appsignal.com/redirect-to/app?to=integrations) and the "repo url" from the selected repository will be stored automatically. With the GitHub integration you'll also be able to send incidents to GitHub directly from AppSignal.
```
